osu.Game/Overlays/Chat/DrawableChannel.cs

```diff
@@ -220,5 +327,53 @@ public DaySeparator(DateTimeOffset time)
                 };
             }
         }
+
+        private class MentionNotification : SimpleNotification
```

Comment:
should be two separate classes. a "chanel" isn't a "username" and you're conditional-ing every line of the class.